src/libraries/System.Net.Http.Json/tests/FunctionalTests/HttpClientJsonExtensionsTests.cs

```diff
@@ -413,6 +470,54 @@ public async Task GetFromJsonAsync_EnforcesTimeout(bool useDeleteAsync, bool slo
                 Exception ex = await Assert.ThrowsAsync<TaskCanceledException>(() =>
                     useDeleteAsync ? client.DeleteFromJsonAsync<string>(uri) : client.GetFromJsonAsync<string>(uri));
 
+#if NETCORE
```

Comment:
Did you mean NETCOREAPP?
```suggestion
#if NETCOREAPP
```

Reply:
It was based on a similar test in the same file, for the `GetFromJsonAsync` call—using the same preprocessor directive:

https://github.com/dotnet/runtime/blob/f016dc4b7cfc306d2d5e29c49e5ccd2d2916d466/src/libraries/System.Net.Http.Json/tests/FunctionalTests/HttpClientJsonExtensionsTests.cs#L416-L419

Reply:
Should I update both?

Reply:
I've updated this in ad465c8.